woocommerce/class-sv-wc-plugin-compatibility.php

```diff
@@ -491,7 +491,22 @@ protected static function is_wc_subscriptions_version( $version ) {
 		return $version === $subscriptions_version || ( $subscriptions_version && version_compare( $version, $subscriptions_version, '=' ) );
 	}
 
+	/**
+	 * Determines whether HPOS is enabled.
+	 *
+	 * @link https://woocommerce.com/document/high-performance-order-storage/
+	 *
+	 * @since x.y.z
+	 *
+	 * @return bool
+	 */
+	public static function is_hpos_enabled() {
 
+		$custom_orders_table_controller = \Automattic\WooCommerce\Internal\DataStores\Orders\CustomOrdersTableController::class;
+
+		return class_exists( $custom_orders_table_controller )
+			&& 'yes' === get_option( $custom_orders_table_controller::CUSTOM_ORDERS_TABLE_USAGE_ENABLED_OPTION );
+	}
```

Comment:
why we went with this solution and not the one described in https://github.com/woocommerce/woocommerce/wiki/High-Performance-Order-Storage-Upgrade-Recipe-Book#detecting-whether-hpos-tables-are-being-used-in-the-store ?